kornia/augmentation/container/augment.py

```diff
@@ -181,11 +181,17 @@ def inverse(  # type: ignore
         """
         if data_keys is None:
             data_keys = cast(List[Union[str, int, DataKey]], self.data_keys)
+
+        _data_keys = list(DataKey.get(inp) for inp in data_keys)
```

Comment:
```suggestion
        _data_keys: List[DataKey] = [DataKey.get(inp) for inp in data_keys]
```
https://deepsource.io/gh/kornia/kornia/run/c16adc3c-beee-4661-8387-49948b333146/python/PTC-W0015/